master/security/namespace-policy.md

```diff
@@ -59,6 +59,29 @@ spec:
       ports:
       - 6379
 ```
+
+#### Restrict label assignment with namespace selectors
+
```

Comment:
You can apply network policies using selectors that match labels on the endpoint, the endpoint namespace, or the endpoint service account. If you specify selectors based on the endpoint namespace, you can apply Kubernetes RBAC to limit which users are allowed to apply labels. In the following example, users in the **development** environment are allowed to communicate only with endpoints in that same namespace.

Reply:
@bmckercher123 it's hard for me to pick out the differences between your suggestion and what is written here. What are the things you think should be changed?